Doc set for VMFR (#110937)
openshift/openshift-docs: modules/oadp-vmfr-creating-vmbd.adoc

// Module included in the following assemblies:
//
// backup_and_restore/application_backup_and_restore/oadp-vmfr/oadp-using-vmfr.adoc

:_mod-docs-content-type: PROCEDURE
[id="oadp-vmfr-creating-vmbd_{context}"]
= Creating a VirtualMachineBackupsDiscovery CR

[role="_abstract"]

> Create a `VirtualMachineBackupsDiscovery` (VMBD) custom resource (CR) to identify which Velero backups contain a specified virtual machine. This helps you to locate available backups before performing a file-level restore.

Reviewer: ```suggestion
Create a `VirtualMachineBackupsDiscovery` (VMBD) custom resource (CR) to identify which Velero backups contain a specified virtual machine (VM). You can locate available backups before performing a file-level restore.
```

After you create a VMBD CR, the CR undergoes the following phases:

* The initial phase for the CR is `New`.
* The controller compiles candidate backups and verifies VM presence in each backup.
* Upon successful discovery, the `status.phase` field of the VMBD CR is updated to `Completed`.

> [NOTE]

Reviewer: ```suggestion
[IMPORTANT]
```

====

> All VMFR custom resources must be created in the protected namespace, which is `openshift-adp` by default.

Reviewer: ```suggestion
Create all VMFR custom resources in the protected namespace, which is `openshift-adp` by default.
```
